This patch is HACK to make kdebindings build. The resulting
PhononSmoke is useless probably.
--- a/smoke/CMakeLists.txt
+++ b/smoke/CMakeLists.txt
@@ -3,7 +3,7 @@
 OPTION(ENABLE_QTSCRIPT_SMOKE "build QtScriptSmoke" ON)
 OPTION(ENABLE_QTUITOOLS_SMOKE "build QtUiToolsSmoke" ON)
 OPTION(ENABLE_QTTEST_SMOKE "build QtTestSmoke" ON)
-OPTION(ENABLE_PHONON_SMOKE "build PhononSmoke" OFF)
+OPTION(ENABLE_PHONON_SMOKE "build PhononSmoke" ON)
 OPTION(ENABLE_QSCI_SMOKE "build QsciSmoke" ON)
 OPTION(ENABLE_QWT_SMOKE "build QwtSmoke" OFF)
 OPTION(ENABLE_KDE_SMOKE "build KDESmoke" ON)
--- a/smoke/phonon/CMakeLists.txt
+++ b/smoke/phonon/CMakeLists.txt
@@ -43,25 +43,25 @@
 
 set(smokephonon_LIB_SRCS ${CMAKE_CURRENT_BINARY_DIR}/smokedata.cpp
 	${CMAKE_CURRENT_BINARY_DIR}/x_1.cpp
-		${CMAKE_CURRENT_BINARY_DIR}/x_2.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_3.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_4.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_5.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_6.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_7.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_8.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_9.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_10.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_11.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_12.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_13.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_14.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_15.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_16.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_17.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_18.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_19.cpp
-                ${CMAKE_CURRENT_BINARY_DIR}/x_20.cpp
+#		${CMAKE_CURRENT_BINARY_DIR}/x_2.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_3.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_4.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_5.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_6.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_7.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_8.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_9.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_10.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_11.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_12.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_13.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_14.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_15.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_16.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_17.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_18.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_19.cpp
+#                ${CMAKE_CURRENT_BINARY_DIR}/x_20.cpp
 )
 
 IF(CMAKE_CXX_FLAGS MATCHES "-fvisibility")
